### PR TITLE
bumping data version to 7.2.5

### PR DESCRIPTION
**Pull request by rheacangeo**
## Purpose

We are updating the data version, because this one is the latest one generated that is running and succeeding in jenkins. 

## Code changes:

- Makefile indicates data version is 7.2.5
- updating the CS_Profile_2d test code to work with a slight change to the serialization statements in fv3gfs-fortran

### CODE REVIEW

File: tests/translate/translate_remap_profile_2d.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ def compute(self, inputs):
         inputs["i1"] = self.grid.global_to_local_x(inputs["i1"] - 1)
         inputs["i2"] = self.grid.global_to_local_x(inputs["i2"] - 1)
         inputs["jslice"] = slice(0, 1)
-
+        if "qs" not in inputs:
+            inputs["qs"] = utils.make_storage_from_shape(self.maxshape)
```
Comment (ofuhrer) [inline tests/translate/translate_remap_profile_2d.py:67]: Makes perfect sense, since `remap_profile.compute()` expects `qs` as one of the inputs.

What I'm not sure how we'll deal with in the long-run, is that we actually don't want to call `make_storage_from_shape` in any of the compute methods when running the dycore, e.g.
```
    qs = utils.make_storage_from_shape(pe1.shape, origin=(0, 0, 0))
```
in `mapn_tracer.compute()`, but this is not related to this PR.

**Review by ofuhrer — APPROVED** [review]
Thanks. LGTM!

**Comment by rheacangeo** [comment]
launch jenkins

File: tests/translate/translate_remap_profile_2d.py (hunk 1 above)
Comment (rheacangeo) [reply tests/translate/translate_remap_profile_2d.py:67]: agreed! we need to get rid of those as much as possible. We are reducing some of them by putting more code into stencils, but there will still be some we may need to push up a few levels and pass through, or something.

**Comment by rheacangeo** [comment]
launch jenkins

**Comment by rheacangeo** [comment]
launch jenkins

**Comment by rheacangeo** [comment]
launch jenkins

**Comment by rheacangeo** [comment]
launch jenkins
